src/api/install.ts

```diff
@@ -531,13 +548,14 @@ async function installInContext (
         skipped: Array.from(installCtx.skipped),
         layoutVersion: LAYOUT_VERSION,
         independentLeaves: opts.independentLeaves,
+        hasPackagesToBuild: hasPackagesToBuild,
```

Comment:
why is this needed? if the array value has any elements, it means there are packages to build